### PR TITLE
Print unhandeled command in FakeExec

### PR DESCRIPTION
**Pull request by ghost**
This makes it easier to figure out what command needs to be mocked or if there are too many executions of a command.

### CODE REVIEW

File: pkg/util/exec/fake_exec.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ type FakeCommandAction func(cmd string, args ...string) Cmd
 
 func (fake *FakeExec) Command(cmd string, args ...string) Cmd {
 	if fake.CommandCalls > len(fake.CommandScript)-1 {
-		panic("ran out of Command() actions")
+		panic(fmt.Sprintf("ran out of Command() actions. Could not handle command [%v]: %v args: %v", fake.CommandCalls, cmd, args))
```
Comment (ashcrow) [inline pkg/util/exec/fake_exec.go]: ```fake.CommandCalls``` should probably be shown as ```%d``` instead of ```%v```.

**Comment by ashcrow** [comment]
One nit but otherwise this looks good and makes sense.

**Comment by swagiaal** [comment]
@ashcrow fixed.. I was being lazy :)

**Comment by ashcrow** [comment]
A ton of errors reported in e2e which I believe are not related... framework, deployment, rc, etc... I recommend rerunning e2e.

**Comment by swagiaal** [comment]
@k8s-bot test this

**Comment by ashcrow** [comment]
@swagiaal magic! :cocktail:

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**